src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -14,6 +13,7 @@ import {
     isIntrinsic,
     ModelType,
     ModelTypeProperty,
+    NeverType,
```

Comment:
what does `NeverType` mean?

Reply:
It's cadl Intrinsic type. The never type indicates the values that will never occur.